DP Mystery Map/Assets/Scripts/questSystem.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PlayerInfo;
+
+public class questSystem : MonoBehaviour
+{
+    public TextAsset openingDialogue;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        if(Player.collectedItems == Item.None)
```

Comment:
This breaks if the player enters the game on floor 2 without any items (i.e., the player saves and loads the game on floor 2 without collecting any items).